File: Day-1/Q1.c
#include <stdio.h>

long sumN(long n){
    return n * (n + 1) / 2;
}

int main(){
    long n;
    printf("Enter an Integer : ");
    scanf("%ld", &n);
    if(n < 1){
        printf("Given input is not a Natural no.");
        return 0;
    }
    printf("Sum = %ld", sumN(n));
    return 0;
}

File: Day-1/Q1.java
import java.util.Scanner;

public class Q1 {
    public static long sumN(long n) {
        return n * (n + 1) / 2;
    }

    public static void main(String[] args) {
        try (Scanner s = new Scanner(System.in)) {
            System.out.print("Enter an Integer : ");
            long n = s.nextLong();
            if (n < 1) {
                System.out.println("Given input is not a Natural no.");
                return;
            }
            
        System.out.println("Sum = " + sumN(n));
        }
    }
}